src/components/Footer/index.js

```diff
@@ -1,15 +1,153 @@
+              </Box>
+              <Box className={classes.meraki}>
+                <img
+                  src={require("./asset/meraki.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+            </Box>
+            <Box sx={{ display: "flex" }}>
+              <Box className={classes.image}>
+                <img
+                  src={require("./asset/facebook.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+              <Box className={classes.image}>
+                <img
+                  src={require("./asset/linkedIn.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+              <Box className={classes.image}>
+                <img
+                  src={require("./asset/twitter.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+            </Box>
+            <Box className={classes.content}>
+              <Typography color="text.primary">
+                © 2022 NavGurukul Foundation for Social Welfare
+              </Typography>
+            </Box>
+          </Grid>
+          <Grid xs={6} md={2} sx={{ pl: "15px" }}>
+            {MenuList("About")}
+          </Grid>
+          <Grid xs={6} md={2}>
+            {MenuList("LearningTracks")}
+          </Grid>
+          <Grid xs={6} md={2} sx={{ pl: "15px" }}>
+            {MenuList("GetInvolved")}
+          </Grid>
+          <Grid xs={6} md={2}>
+            <Typography
+              color="text.primary"
+              sx={{ mt: 4, mb: 1 }}
+              variant="subtitle1"
+              component="div"
+            >
+              Learn on Mobile
+            </Typography>
+            <Box sx={{ display: "flex" }}>
+              <img
+                src={require("./asset/playStore.svg")}
+                alt="play store"
+                loading="lazy"
+              />
+              <Box sx={{ padding: "4px 0 0 10px" }}>
+                <Typography color="text.primary" component="div">
+                  Now on Playstore
+                </Typography>
+              </Box>
+            </Box>
+          </Grid>
+        </Grid>
+        <Divider variant="string" sx={{ pt: "25px" }} />
+        <Box>
+          <Grid container spacing={2} sx={{ m: "30px 0px 30px 0px" }}>
+            <Grid xs={12} md={6} sx={{ pl: { sm: "none", md: "10px" } }}>
+              <Typography color="text.primary">
+                Legal & Privacy Policy
+              </Typography>
+            </Grid>
+            <Grid
+              xs={12}
+              md={6}
+              sx={{
+                pr: { sm: "none", md: "17px" },
+              }}
+            >
+              <Typography
+                color="text.primary"
+                sx={{ textAlign: { sm: "none", md: "right" } }}
```

Comment:
`none` is not a valid value for `textAlign` in CSS.  (I'm not positive about MUI.)  Did you want to use `left` instead?
```suggestion
                sx={{ textAlign: { sm: "left", md: "right" } }}
```

Reply:
Yes.